docs/Packaging-an-iOS-component.md

```diff
@@ -0,0 +1,54 @@
+---
+id: packaging-an-iOS-component
+title: Packaging an iOS component
+layout: docs
+category: Guides (iOS)
+permalink: docs/packaging-an-iOS-component.html
+next: native-modules-android
+---
+
+You can also inject native modules into React Native by creating a project in Objective-C and publish it as npm package.
+
+### Creating the package
+
+ * Create a new Xcode project and fill out the product name, organization name and organization identifier. Ensure that the name of your project is prefixed with RN.
+
+ * `RN` is a prefix mainly used to indicate that it is a
+react-native standard library.It is not required, but it is encouraged.
+
+ * `cd` to the project directory and run `npm init` and fill out your information such as package-name,version and so on.
+
+ * `npm install react-native --save-dev`, then open up `package.json`
+   and  add  `peerDependencies: { "react-native": "*" }`.
+
+ * Now we need to tell Xcode where it can find the React Native source. Open up the project, add React to your Header Search Paths.
+
+![header-paths-2](img/5-header-search-paths.png)
+
+ * The highlighted one is for development.The package will be available in `node_modules` directory of the project that is published as a package.We have to add that package to the new project and link it.
```

Comment:
"...in the `node_modules`..."

space after the period after the word "package."